### DISCUSSION TITLE
Lag Issues

### DISCUSSION THREAD

**Discussion by calebjamesross** [samemory/homebridge-eufy-security-control#37] [General]
I am having some serious lagging issues with notifications and sensors opening/closing. Any idea on how to solve this? 

Thank you so much for this plugin!

**Comment by schliemann** [comment]
How much lag? How many entry sensors do you have? Do you receive the eufy app push notifications without lag?

Right now the only automatic state updates are from push notifications. But it might make sense to change this to also support polling.
